### DISCUSSION TITLE
Error: https://registry.npmjs.org/xxx.tgz: Request failed "503 Service Unavailable"

### DISCUSSION THREAD

**Discussion by guy-frontegg** [npm/feedback#970] [Registry] [answered]
Hello All,

For the past week or so we are experiencing the error written on the title.
The errors are for mainly our private registry on Npm.
We get it from GitHub actions and our local workstations. I opened a ticket in NPM and Github without any answer for now.

Anyone encounter such issues?

**Comment by feelepxyz** [comment, marked as answer]
@guy-frontegg sorry for missed comms on this. This is being worked on but the team is still figuring out the root cause.

**Reply by janhommes** [reply]
We are having this issue and it is blocking our releases for a couple of weeks. Are you still trying to find the root cause?

**Comment by matt-carpenter** [comment]
I can't deploy my builds because of this error currently. I am deploying through circle ci from our private registry. Is there any update on this?

**Comment by reey** [comment]
Is there any update available regarding this? We are facing the same issue and opened a [ticket](https://enterprise.githubsupport.com/hc/en-us/requests/2274982) but we didn't had any progress on this.

**Comment by tiagonbotelho** [comment]
Hey everyone, apologies for the radio silence in this discussion and the trouble caused by this incident. We have since deployed a change that we believe should fix this specific issue, if you are still observing the same issue, please let us know so that we can continue investigating.

**Reply by reey** [reply]
@tiagonbotelho we are sadly still experiencing this issue, still the same `503 Service Unavailable`.

**Reply by tiagonbotelho** [reply]
Sorry to hear that, could you perhaps share your logs and request-ids if available with us through your support ticket, so that we can troubleshoot this issue further?

Thanks in advance

**Reply by reey** [reply]
@tiagonbotelho we've shared a debug log from today via the support ticket.
Regarding the request-ids, how do we get them?

Thanks

**Reply by jumoel** [reply]
Hey @reey :wave:

Just wanted to let you know that we're looking at the logs you attached.

The ticket you opened was through GitHub support. Can I get you to open the same ticket through https://www.npmjs.com/support instead? That means it'll be easier for us to keep track of. Thanks!

**Reply by reey** [reply]
Hi @jumoel 

We've actually raised the ticket via https://www.npmjs.com/support the confirmation email included the ticketId: 2274982, maybe that helps you?

Thanks for the support.

**Reply by reey** [reply]
@tiagonbotelho @jumoel Do you maybe have an update regarding this?
We are facing this issue now since a month and there was not really any progress on the ticket yet.. 😞
Any info like a root cause or especially a workaround would be really helpful for us..

Thanks!
